sdk/core/src/CommonNetworkMember.ts

```diff
@@ -632,6 +635,16 @@ export class CommonNetworkMember {
 
     const { userPoolId, clientId } = CommonNetworkMember.setEnvironmentVarialbles(options)
 
+    if (messageParameters) {
+      const customMessageTemplateService = new CustomMessageTemplatesService(options)
```

Comment:
But we already have `this._customMessageTemplateService` (which is not used anywhere)